For gcc6 a string literal must be separated from a macro variable
by at least one space.

--- plugins/latex/xp/ie_exp_LaTeX.cpp	2013-04-07 15:53:03.000000000 +0200
+++ plugins/latex/xp/ie_exp_LaTeX.cpp	2016-09-18 16:09:08.087076449 +0200
@@ -1329,7 +1329,7 @@
 	m_pie->write(sBuf.c_str(),sBuf.size());
 }
 
-#define SUB(a,who) case a: subst = "\\(\\"who"\\)"; return true;
+#define SUB(a,who) case a: subst = "\\(\\" who "\\)"; return true;
 #define SUBd(a,who) case a: subst = who; return true;
 static bool _convertLettersToSymbols(char c, const char *& subst)
 {
--- plugins/xslfo/xp/ie_exp_XSL-FO.cpp	2013-04-07 15:53:03.000000000 +0200
+++ plugins/xslfo/xp/ie_exp_XSL-FO.cpp	2016-09-18 16:13:55.980997056 +0200
@@ -1451,7 +1451,7 @@
 	{ \
 		UT_UTF8String esc = szValue; \
 		esc.escapeXML(); \
-		buf += " "x"=\""; \
+		buf += " " x "=\""; \
 		buf += esc.utf8_str(); \
 		buf += "\""; \
 	}
